SPECS/cni-plugins/cni-plugins.spec

```
@@ -13,6 +13,8 @@ URL:            https://github.com/containernetworking/plugins
 Source0:        %{name}-%{version}.tar.gz
 BuildRequires:  golang >= 1.5
 
+Provides:      kubernetes-cni
```

```suggestion
Provides:       kubernetes-cni
```
Please fix the linter errors surrounding the provides. I think there's a newline delta here too.

Please ignore the %define linter warning since I think we prefer having the %defines at the top of the file

---

Amended in a commit! Thanks!